### DISCUSSION TITLE
Add Sort by Date Created in Photopea File Browser

### DISCUSSION THREAD

**Discussion by UltimateWishub** [photopea/photopea#5742] [Ideas]
![image](https://github.com/photopea/photopea/assets/60529992/29d0fcb7-eed2-4e28-aee0-0e6194ac0c17)

**Comment by photopea** [comment]
When there is "Date", it is sorted by the date of the last modification of each file. You can click the arrow on the left from the "Date" button to switch between "ascending" and "descending" sorting.

We do not allow sorting by the date of a creation, only by the date of the last modification.

**Reply by UltimateWishub** [reply]
That's why I'm asking if sort by date created can be added

**Reply by photopea** [reply]
Currently, browsers do not tell websites any creation date, we can access only the "lastModified" date - https://w3c.github.io/FileAPI/#dfn-file .
